src/components/SourceFooter.css

```diff
@@ -12,50 +12,39 @@
   user-select: none;
 }
 
-html:not([dir="rtl"]) .source-footer .command-bar {
-  float: right;
+.source-footer__command-bar {
+  display: flex;
+  padding: 8px 0.7em;
 }
 
-html[dir="rtl"] .source-footer .command-bar {
-  float: left;
-}
-
-.source-footer .command-bar * {
+.source-footer__command-bar * {
   -moz-user-select: none;
   user-select: none;
 }
 
-.command-bar > span {
+.source-footer__command-bar > .source-footer__command-bar__action {
   cursor: pointer;
-  width: 1em;
-  height: 1.1em;
-  display: inline-block;
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   transition: opacity 200ms;
+  border: none;
+  background: transparent;
 }
 
-html:not([dir="rtl"]) .command-bar > span {
-  margin-right: 0.7em;
-}
-
-html[dir="rtl"] .command-bar > span {
-  margin-left: 0.7em;
+.source-footer__command-bar .prettyPrint.pretty {
+  stroke: var(--theme-highlight-blue);
 }
 
-.source-footer .prettyPrint.pretty {
-  stroke: var(--theme-highlight-blue);
+:root.theme-dark .source-footer__command-bar > .source-footer__command-bar__action {
+  fill: var(--theme-body-color);
 }
```

Comment:
This was necessary here since originally it pulled from the code for the sidebar's command bar. Due to the rename of classes to scope it down, this is now required here.